package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ui-box",
-  "version": "4.1.0",
+  "version": "4.2.0",
```

Comment:
FYI No need to bump this version here, as that will be handled automatically during releases.